src/deployment/mod.rs

```diff
@@ -499,6 +531,26 @@ pub fn apply_on_chain_deployment(
         let mut batch = Vec::new();
         for transaction in batch_spec.transactions.iter() {
             let tracker = match transaction {
+                TransactionSpecification::BtcTransfer(tx) => {
+                    let account = btc_accounts_lookup.get(&tx.expected_sender).unwrap();
+                    let (secret_key, _public_key) = get_btc_keypair(account);
+                    let res = bitcoin_deployment::send_transaction_spec(tx, &secret_key);
+                    println!("=> {:?}", res);
+
+                    // let name = format!(
+                    //     "Send {} Btc from {} to {}",
+                    //     tx.amount,
+                    //     tx.expected_sender.to_string(),
+                    //     tx.recipient
+                    // );
+                    // let check = TransactionCheck::BtcTransfer();
+                    // TransactionTracker {
+                    //     index,
+                    //     name: name.clone(),
+                    //     status: TransactionStatus::Encoded(transaction, check),
+                    // }
```

Comment:
Can this be removed?

Reply:
Addressed in https://github.com/hirosystems/clarinet/pull/402/commits/ce821313ed66b9190826dd147cdcc477cf6908f0